File: 3way/problem1/kthFactor.c
int kthFactor(int n, int k) {
    int i = 1;
    for(i; i <= n; i++){
        if(n % i == 0){
            k--;
        }
        if(k == 0){
            return i;
        }
    }
    return -1;
}

File: 3way/problem1/kthFactor.js
/**
 * @param {number} n
 * @param {number} k
 * @return {number}
 */
var kthFactor = function(n, k) {
    
    for(var i = 1; i <= n; i++){
        if(n % i == 0) k--;
        if(k == 0) return i;
    }
    
    return -1;

};
